<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="messageLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendFileButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send File Socket</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMessageButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="uploadStatus">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Record/*Sender</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendToPhpButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>send php file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startServerButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopServerButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chatTextBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rateStatus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileSize">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
